.github/workflows/publish.yml

```
@@ -0,0 +1,265 @@
+name: Publish Docker image, Helm chart and Wire build
+
+on:
+  push:
+    branches: [master, dev, edge, avs, mobile, acc, stefan]
```

```suggestion
    branches: [master, dev, edge, stefan]
```

Those are retired envs, now is a good time to sunset them and start fresh :)

---

Done in f0128a2fc069e442e2fdb919acdbcc23abb5b3d0